internal/pkg/service/stream/sink/router/router.go

```diff
@@ -40,23 +47,43 @@ type Router struct {
 
 	// wg waits for all writes/goroutines
 	wg sync.WaitGroup
+
+	// OTEL metrics
+	meters *meters
```

Comment:
Is it meters? Isn't this rather `metrics`?

Reply:
OpenTelemetry is using `meter`/`meters` wording, see GO pkgs.

Reply:
Ok, but this is official OTEL Go package:
![image](https://github.com/user-attachments/assets/4add5a8c-2de7-4f42-b651-92d2df5dfefa)

Of course, feel free to rename it.